ee/tables/dataflattentable/tables.go

```diff
@@ -165,16 +173,43 @@ func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) (
 			}
 		}
 	}
+
+	for _, rawdata := range requestedRawDatas {
+		for _, dataQuery := range tablehelpers.GetConstraints(queryContext, "query", tablehelpers.WithDefaults("*")) {
+			subresults, err := t.generateRawData(ctx, rawdata, dataQuery, append(flattenOpts, dataflatten.WithQuery(strings.Split(dataQuery, "/")))...)
+			if err != nil {
+				t.slogger.Log(ctx, slog.LevelInfo,
+					"failed to get data for raw_data",
+					"err", err,
+				)
+				continue
+			}
+
+			results = append(results, subresults...)
+		}
+	}
+
 	return results, nil
 }
 
-func (t *Table) generatePath(ctx context.Context, filePath string, dataQuery string) ([]map[string]string, error) {
-	flattenOpts := []dataflatten.FlattenOpts{
-		dataflatten.WithSlogger(t.slogger),
-		dataflatten.WithNestedPlist(),
-		dataflatten.WithQuery(strings.Split(dataQuery, "/")),
+func (t *Table) generateRawData(ctx context.Context, rawdata string, dataQuery string, flattenOpts ...dataflatten.FlattenOpts) ([]map[string]string, error) {
+	data, err := t.flattenBytesFunc([]byte(rawdata), flattenOpts...)
+	if err != nil {
+		t.slogger.Log(ctx, slog.LevelInfo,
+			"failure parsing raw data",
+			"err", err,
+		)
+		return nil, fmt.Errorf("parsing data: %w", err)
 	}
 
+	rowData := map[string]string{
+		"raw_data": rawdata,
```

Comment:
not sure if we should actually include this, can return a lot of data

I suppose you could always just select columns you want in query.

Reply:
Sadly, we have to. If you don't, sqlite will filter out the rows. I'm not sure if we have another option today

Reply:
Er, at least for the case where `raw_data` is provided. We should skip for file ones